exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -0,0 +1,30 @@
+public class CalculatorConundrum {
+    private final String answer;
+
+    public CalculatorConundrum(int operand1, int operand2, String operation) {
+        int result;
+        try {
```

Comment:
In Java, you want a try-catch block to be scoped at the correct level. As far as I can tell, the correct level is only surrounding the division case, but depending on how we structure the code, it may have to go around a generic operation setup.